client/src/pages/private/ParticipantTable.js

```diff
@@ -387,25 +285,28 @@ export default () => {
     }
   };
 
-  const forceReload = async (currentPagination) => {
-    if (!tabValue) return;
-    const currentPage = currentPagination.currentPage;
+  const forceReload = async () => {
+    if (!reducerState.tabValue) return;
+    const currentPage = reducerState.pagination?.currentPage || 0;
     setLoadingData(true);
     const { data, pagination } = await fetchParticipants(
       currentPage * pageSize,
-      locationFilter,
-      fsaFilter,
-      lastNameFilter,
-      emailFilter,
-      order.field,
-      order.direction,
-      siteSelector,
-      tabs[tabValue].statuses
+      reducerState.locationFilter,
+      reducerState.fsaFilter,
+      reducerState.lastNameFilter,
+      reducerState.emailFilter,
+      reducerState.order.field,
+      reducerState.order.direction,
+      reducerState.siteSelector,
```

Comment:
I'd be partial to destructuring these values at the start of the function to avoid the repitition, bit of a nit though.

Reply:
I played around with deconstructing these, but I ran into an issue where the useEffect required adding the entire reducer state to the it's listener. The result was an infinite loop. 

I think I am going to hold off implementing this for now, but I am all onboard with refactoring the useEffect later.

Reply:
Oooh gotcha, yeah let's save this for another day then 👍